gammapy/datasets/make.py

```diff
@@ -64,3 +70,90 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def generate_observation_table(observatory, n_obs):
+    """Generate an observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory : string
+    	name of the observatory
+    n_obs : integer
+    	number of observations for the obs table
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    astro_table = Table()
```

Comment:
Is there a reason why you can't just instantiate an `ObservationTable` here and then fill it?
Does it do some checking in the constructor that prevents that pattern of creating it?
If no, I think I'd prefer that (a bit simpler).